test/ui-test/src/screens/StatusWalletScreen.py

```diff
@@ -193,11 +208,36 @@ def delete_saved_address(self, name: str):
         
         assert found != -1, "saved address not found"
         
-        # More icon is at index 2 
+        # More icon is at index 2
+        time.sleep(1)
         click_obj(list.itemAtIndex(found).components.at(2))
         
         click_obj_by_name(SavedAddressesScreen.DELETE.value)
         click_obj_by_name(SavedAddressesScreen.CONFIRM_DELETE.value)
+    
+    def toggle_network(self, network_name: str):
+        click_obj_by_name(MainWalletScreen.NETWORK_SELECTOR_BUTTON.value)
+
+        list = get_obj(NetworkSelectorPopup.LAYER_1_REPEATER.value)
+        for index in range(list.count):
+            if list.itemAt(index).objectName == network_name:
+                click_obj(list.itemAt(index))
+                click_obj_by_name(MainWalletScreen.ACCOUNT_NAME.value)
+                time.sleep(2)
+                return
+        
+        assert False, "network name not found"
+
+        
+    def verify_positive_balance(self, symbol: str):
+        list = get_obj(AssetView.LIST.value)
+        for index in range(list.count):
+            if list.itemAtIndex(index).objectName == symbol:
+                balance = list.itemAtIndex(index).children.at(2).text
+                assert balance != f"0 {symbol}", "balance is not positive"
```

Comment:
We could print the balance found as well maybe
```suggestion
                assert balance != f"0 {symbol}", "balance is not positive. Balance: "str(balance)
```